community/io/src/test/java/org/neo4j/adversaries/ClassGuardedAdversary.java

```diff
@@ -74,7 +74,8 @@ private boolean calledFromVictimClass()
         StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
         for ( StackTraceElement element : stackTrace )
         {
-            if ( victimClasses.contains( element.getClassName() ) )
+            Class<?> elementClass = classForName( element.getClassName() );
```

Comment:
what do you think if instead of doing `Class.forName` each time we checking for victim class we will do vice versa: will store in victim classes names of the classes and just compare names during calls?